.github/workflows/linode_stage.yaml

```
@@ -19,7 +19,7 @@ jobs:
     - name: Run linter
       run: yarn run lint
     - name: Run build task
-      run: NODE_ENV=production yarn run build
+      run: NODE_ENV=production SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }} SENTRY_AUTO_RELEASE=true yarn run build
```

Should `SENTRY_AUTO_RELEASE` be used with the env variable? 
```suggestion
      run: NODE_ENV=production SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }} SENTRY_AUTO_RELEASE=${{ secrets.SENTRY_AUTH_TOKEN }} yarn run build
```